test/falcor/error/index.js

```diff
@@ -281,7 +281,7 @@ describe("Error", function() {
             subscribe(noOp, doneOnError(done), errorOnCompleted(done));
     });
 
-    it('should allow for dataSources to immediately throw an error (set)', function(done) {
+    xit('should allow for dataSources to immediately throw an error (set)', function(done) {
```

Comment:
Did you still want this skipped?

Reply:
well it's actually failing: https://github.com/Netflix/falcor/issues/880

Reply:
Cool, LGTM